tsconfig.json

```
@@ -0,0 +1,5 @@
+{
+	"compilerOptions": {
+		"strictNullChecks": true
```

```suggestion
		"strict": true
```

to get all the strict goodness.